Taskfile.yml

```diff
@@ -191,6 +191,7 @@ tasks:
       END_BLOCK: '{{.END_BLOCK}}'
       LABELS: '{{.LABELS | default ""}}'
       BENCHMARK_OUTPUT_FILE: '{{.BENCHMARK_OUTPUT_FILE | default ""}}'
+      METRICS_ENABLED: '{{.METRICS_ENABLED | default ""}}'
```

Comment:
This results in some subtle behavior that we should prefer to make explicit.

By setting the default to "", this variable is falsy and will not be appended in the benchmark script, which means the default of "" subtly results in defaulting to enabled, but that's not obvious at first glance.

Could we make this explicit by making the default `true` in both places rather